docs/develop/01_sdk/01_quickstart.md

```diff
@@ -5,39 +5,40 @@ title: Quickstart
+import TsJsBlock from '@site/src/components/TsJsBlock';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 import PrintHelloWorld from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/getting_started/01_print_hello_world.ts';
-import Connect from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/getting_started/02_connect.ts';
+import ConnectSpirit from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/getting_started/02_connect_spirit.ts';
+import ConnectPere from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/getting_started/02_connect_pere.ts';
 import FetchDid from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/getting_started/03_fetch_did.ts';
 import FetchEndpoints from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/getting_started/04_fetch_endpoints.ts';
 import FetchEndpointData from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/getting_started/05_fetch_endpoint_data.ts';
 import VerifyCredential from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/getting_started/06_verify_credential.ts';
 import Disconnect from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/getting_started/07_disconnect.ts';
 
-The following guide will give you a starting point to begin with KILT.
-You will learn how to:
+Get started with KILT by following this guide.
+You'll learn to:
 
-1. Import the **KILT SDK** into a project
+1. Import the **KILT SDK** into your project
 2. Connect to the **KILT blockchain**
-3. Query a **web3name** to get its **DID**
-4. Verify a **credential**, available via a **DID service**
+3. Query a **web3name** to retrieve its **DID**
+4. Verify a **credential** using a **DID service**
 
-After completing the quickstart guide, you should have gained a better understanding of KILT through hands-on experience.
-The guide requires some experience with javascript and command-line tools.
-We will recommend guides to other tutorials to dive deeper into some of the topics.
+This quickstart guide provides hands-on experience to enhance your understanding of KILT.
+Basic knowledge of JavaScript and command-line tools is recommended.
+For more in-depth tutorials, we'll suggest additional guides.
 
 ## Setup
 
-We will focus on creating a new project from scratch, which will require a little setup.
-First, we need to create a new project in a new directory.
-For this, we run `mkdir kilt-rocks && cd kilt-rocks`.
+Let's start by creating a new project from scratch.
+Create a new project in a fresh directory by running `mkdir kilt-rocks && cd kilt-rocks`.
 
 <Tabs groupId="ts-js-choice">
   <TabItem value='ts' label='Typescript' default>
 
-  From inside the `kilt-rocks` project directory, install the **KILT SDK**, **Ts-node**, **Axios** and **Typescript**:
+  From inside the `kilt-rocks` project directory, install the **KILT SDK**, **Typescript**, **ts-node** and **Axios**:
```

Comment:
Should we then also change the order of the installation in the `npm install` command below?